kalite/distributed/static/js/distributed/user/views.js

```diff
@@ -89,6 +97,11 @@ window.LoginView = BaseView.extend({
     handle_login: function(response) {
         if (response.status == 200) {
             this.trigger("login_success");
```

Comment:
I am not sure what that side effect might be. This is being used to close the modal upon success, which is a reasonable UI expectation regardless of whether the page subsequently redirects (from a user point of view the modal dismissing is a good indicator of the login success, even if momentarily).

Unless this event was being used to trigger some server side logging, I don't know how this could possibly result in an unsafe expectation.